### ISSUE TITLE
The sampled images contain noise.

### ISSUE THREAD

**Issue by summergray0608-stack** [mirthAI/Fast-DDPM#36]
Hello, thank you for your work!I encountered the following issue while training Fast-DDPM on the BraTS 2023 dataset: following the recommendations in the paper and the README, I used a learning rate of 1e-4 and 10 diffusion timesteps, and sampled images at 400,000 training steps. Although the sampled images show brain contours and some details, they contain noticeable noise of varying sizes.What might be the possible causes for this? Thank you for your advice.

<img width="240" height="240" alt="Image" src="https://github.com/user-attachments/assets/3320f134-af8f-435c-b17d-fb4ef1cf038c" />

<img width="240" height="240" alt="Image" src="https://github.com/user-attachments/assets/8d7e7851-50f8-4a40-8c9b-a2b3b0dc4e70" />

<img width="240" height="240" alt="Image" src="https://github.com/user-attachments/assets/20f31ef5-37d6-439e-a305-61ef995ece71" />
